lib/plug_devise_session/rememberable.ex

```diff
@@ -36,6 +28,23 @@ defmodule PlugDeviseSession.Rememberable do
     signing_salt: "signed cookie"
   ]
 
+  @doc """
+  Removes the remember user cookie.
+
+  ## Options
+
+    * `:domain` - domain the remember user cookie was issued in.
+  """
+  @spec forget_user(Plug.Conn.t(), scope, domain: String.t(), max_age: integer) :: Plug.Conn.t()
+  def forget_user(conn, scope \\ :user, opts \\ []) do
+    cookie_opts =
+      [http_only: true]
+      |> Keyword.merge(opts)
+      |> Keyword.take([:domain, :http_only])
```

Comment:
If only `:domain, :http_only` is taken from options why not to explicitly build `cookie_opts`?

Reply:
If you mean why not do this:

```elixir
cookie_opts =
  [
    domain: Keyword.get(opts, :domain),
    http_only: Keyword.get(opts, :http_only, true)
  ]
```

Then the answer is in order not to include keys that were not set in `opts` at all. With the solution above, `domain` would be set to `nil`; with `Keyword.take/2` there would be no `domain` key in the keyword list.